lib/dashman_functions.sh

```diff
@@ -561,7 +561,7 @@ update_dashd(){
             # patch it -----------------------------------------------------------
 
             pending "  --> updating crontab... "
-            (crontab -l 2>/dev/null | grep -v sentinel.py ; echo "* * * * * cd $INSTALL_DIR/sentinel && venv/bin/python bin/sentinel.py  2>&1 >> sentinel-cron.log") | crontab -
+            (crontab -l 2>/dev/null | grep -v sentinel.py ; echo "* * * * * cd $INSTALL_DIR/sentinel && venv/bin/python bin/sentinel.py 2>>sentinel-cron.log >> sentinel-cron.log") | crontab -
```

Comment:
Thanks for these patches! I'll be free to give dashman some attention soon and will review your submissions then.

In the meantime, why doesn't `2>&1` fold STDERR into STDOUT like I'm expecting it to?
If you want to chat about it, here's a discord link: http://dashchat.org/

Reply:
WOW! I learned something new today. Thank you!

I've occasionally gotten this wrong without realizing why, reading it to myself as:
"take the output of moo, fold STDERR into STDOUT, then redirect STDOUT into cow"
when it's actually:
"take the output of moo, redirect STDOUT into cow, _with STDERR folded into STDOUT_"

I had no idea I didn't know this important syntax! THANK YOU!